src/main/java/teammates/ui/request/FeedbackSessionBasicRequest.java

```diff
@@ -127,7 +128,9 @@ public void setPublishedEmailEnabled(boolean publishedEmailEnabled) {
     @Override
     public void validate() throws InvalidHttpRequestBodyException {
         assertTrue(instructions != null, "Instructions cannot be null");
+        assertTrue(!Objects.isNull(submissionStartTimestamp), "Submission Start Time Stamp cannot be null");
```

Comment:
can we replace this with `submissionStartTimestamp != null`

Reply:
The `submissionStartTimestamp` won't be able to be `null` here because it's of type `long` (not `Long`), which doesn't support null values